cmd/main.go

```diff
@@ -23,31 +22,14 @@ func print_all_variables() {
 
 func main() {
 	print_all_variables()
-	token := os.Getenv("GITHUB_TOKEN")
-	githubRepositoryName := strings.Join(strings.Split(os.Getenv("GITHUB_REPOSITORY"), "/")[1:], "")
-	repoOwner := os.Getenv("GITHUB_REPOSITORY_OWNER")
-	eventName := os.Getenv("GITHUB_EVENT_NAME")
-	pullRequestNumber, err := strconv.Atoi(os.Getenv("GITHUB_PR_NUMBER"))
-
-	if err != nil {
-		log.Fatal(err)
-	}
-
-	params := github_connection.GithubConnectionParams{
-		RepositoryName:    githubRepositoryName,
-		Token:             token,
-		RepoOwner:         repoOwner,
-		PullRequestNumber: pullRequestNumber,
-	}
-	githubCodeRepositoryProvider, err := github_connection.NewGithubConnection(params)
+	githubConnection, err := github_connection.NewGithubConnection()
 	if err != nil {
```

Comment:
Code Review 2024-08-18 01:18:33 - New lines 1 to 1:

	githubConnection, err := github_connection.NewGithubConnection()